featurebyte/service/preview.py

```diff
@@ -98,16 +98,24 @@ async def shape(self, preview: FeatureStorePreview) -> FeatureStoreShape:
         FeatureStoreShape
             Row and column counts
         """
-        feature_store, session = await self._get_feature_store_session(
-            graph=preview.graph,
-            node_name=preview.node_name,
-            feature_store_id=preview.feature_store_id,
-        )
-        shape_sql, num_cols = GraphInterpreter(
-            preview.graph, source_type=feature_store.type
-        ).construct_shape_sql(node_name=preview.node_name)
-        logger.debug("Execute shape SQL", extra={"shape_sql": shape_sql})
-        result = await session.execute_query(shape_sql)
+        with timer("Get feature store and session", logger):
```

Comment:
Would it be useful to add a prefix like `shape` to the log message or something in the extra to allow filtering the logs?